src/components/MainPage/Tabs.js

```diff
@@ -36,18 +36,23 @@ const Tabs = ({ activeTab, onTabChange }) => {
       </button>
       <button
         type="button"
-        className={cn("MainPage__tab", { MainPage__tab_active: activeTab === JAILS })}
-        onClick={createOnTabChange(JAILS)}
+        className={cn("MainPage__tab", {
+          MainPage__tab_active: activeTab === JAILS,
+          "MainPage__tab--disabled": isUnified,
+        })}
+        onClick={isUnified ? null : createOnTabChange(JAILS)}
       >
-        Jails
+        Jails{isUnified && " (Not Applicable)"}
       </button>
     </div>
   );
 };
+// This state operates a "unified corrections system," which combines the jail and prison systems. As such, some of the numbers below may include pre-trial populations.
```

Comment:
this comment seems out of place? or maybe just needs to be deleted

Reply:
yeah this was a bad paste, deleted!